dot/network/message.go

```diff
@@ -34,30 +34,31 @@ import (
 
 //nolint
 const (
-	StatusMsgType             = 0
-	BlockRequestMsgType       = 1
-	BlockResponseMsgType      = 2
-	BlockAnnounceMsgType      = 3
-	TransactionMsgType        = 4
-	ConsensusMsgType          = 5
-	RemoteCallRequestType     = 6
-	RemoteCallResponseType    = 7
-	RemoteReadRequestType     = 8
-	RemoteReadResponseType    = 9
-	RemoteHeaderRequestType   = 10
-	RemoteHeaderResponseType  = 11
-	RemoteChangesRequestType  = 12
-	RemoteChangesResponseType = 13
-	ChainSpecificMsgType      = 255
+	StatusMsgType             byte = 0
+	BlockRequestMsgType       byte = 1
+	BlockResponseMsgType      byte = 2
+	BlockAnnounceMsgType      byte = 3
+	TransactionMsgType        byte = 4
+	ConsensusMsgType          byte = 5
+	RemoteCallRequestType     byte = 6
+	RemoteCallResponseType    byte = 7
+	RemoteReadRequestType     byte = 8
+	RemoteReadResponseType    byte = 9
+	RemoteHeaderRequestType   byte = 10
+	RemoteHeaderResponseType  byte = 11
+	RemoteChangesRequestType  byte = 12
+	RemoteChangesResponseType byte = 13
+	ChainSpecificMsgType      byte = 255
 )
 
 // Message interface
 type Message interface {
 	Encode() ([]byte, error)
 	Decode(io.Reader) error
 	String() string
-	Type() int
+	Type() byte
```

Comment:
I'm curious why this is changed from int to byte?

Reply:
the message types only go up to 255 which suggests that it's a byte, and we probably won't have >256 message types so and int is likely unnecessary